<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageOrder</class>
 <widget class="QWidget" name="PageOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QComboBox" name="comboBoxSprayers">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditOrderName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Nazwa zamówienia</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMove">
         <property name="text">
          <string>Przenies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPrint">
         <property name="text">
          <string>Drukuj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Zamknij okno</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
